packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -1,155 +1,72 @@
-import { execSync, ExecSyncOptions } from 'node:child_process';
+import { getPackageManager, packageManagerCreate, packageManagerExec, packageManagerRun, prepareTestApp, setupLocalRegistry } from '@o3r/test-helpers';
+import { ExecSyncOptions } from 'node:child_process';
 import * as fs from 'node:fs';
-import { cpSync, existsSync, mkdirSync, readdirSync, rmSync } from 'node:fs';
+import { cpSync, mkdirSync } from 'node:fs';
 import * as path from 'node:path';
-import getPidFromPort from 'pid-from-port';
-import type { PackageJson } from 'type-fest';
 
 const appName = 'test-sdk';
+const sdkPackageName = '@my-test/sdk';
 const currentFolder = path.join(__dirname, '..', '..', '..', '..');
 const parentFolderPath = path.join(currentFolder, '..');
 const itTestsFolderPath = path.join(parentFolderPath, 'it-tests');
 const sdkFolderPath = path.join(itTestsFolderPath, 'test-sdk');
-const cacheFolderPath = path.join(currentFolder, '.cache', appName);
+const sdkPackagePath = path.join(sdkFolderPath, sdkPackageName.replace(/^@/, ''));
+const execEnv: ExecSyncOptions['env'] = {
+/* eslint-disable @typescript-eslint/naming-convention, camelcase */
+  ...process.env,
+  CI: 'true',
+  JEST_WORKER_ID: undefined,
+  NODE_OPTIONS: '',
+  npm_execpath: undefined,
+  npm_config_user_agent: undefined
+/* eslint-enable @typescript-eslint/naming-convention, camelcase */
+};
 const execAppOptions: ExecSyncOptions = {
   cwd: sdkFolderPath,
-  stdio: 'inherit',
+  stdio: 'pipe',
```

Comment:
Why do you want to `pipe` the stdio per default?
It helps to retrieve the output but it is degrading the console display (if you display it in the end of the execution)

Reply:
when using `inherit` the reporter cannot intercept the output of the commands when there is an exception